src/main/java/uk/gov/hmcts/reform/sscs/evidenceshare/callback/handlers/RoboticsCallbackHandler.java

```diff
@@ -93,47 +98,68 @@ public void handle(CallbackType callbackType, Callback<SscsCaseData> callback) {
             log.info("Is case valid to send to robotics {} for case id {}", isCaseValidToSendToRobotics, callback.getCaseDetails().getId());
 
             if (isCaseValidToSendToRobotics) {
-                updateRpc(callback);
+                updateRpc(callback.getCaseDetails().getCaseData());
                 roboticsService.sendCaseToRobotics(callback.getCaseDetails());
```

Comment:
Should the robotic call be in the mutator of - updateCcdCaseService.updateCaseV2?

Reply:
talked it through with Somesh - verified this setup is correct